app/api/groups/get_user_batch_prefixes.feature

```diff
@@ -0,0 +1,76 @@
+Feature: List user-batch prefixes (userBatchPrefixesView)
+  Background:
+    Given the database has the following table 'groups':
+      | id | name   | grade | type  |
+      | 13 | class  | -2    | Class |
+      | 14 | class2 | -2    | Class |
+      | 21 | user   | -2    | User  |
+    And the database has the following table 'users':
+      | login | group_id |
+      | owner | 21       |
+    And the database has the following table 'group_managers':
+      | group_id | manager_id | can_manage  |
+      | 13       | 21         | memberships |
+    And the database has the following table 'groups_ancestors':
+      | ancestor_group_id | child_group_id |
+      | 13                | 13             |
+      | 13                | 21             |
+      | 14                | 14             |
+      | 21                | 21             |
+    And the database has the following table 'groups_groups':
+      | parent_group_id | child_group_id |
+      | 13              | 21             |
+    And the database has the following table 'user_batch_prefixes':
+      | group_prefix | group_id | allow_new | max_users |
+      | test         | 13       | 1         | 90        |
+      | test1        | 13       | 1         | null      |
+      | test2        | 13       | 0         | 80        |
+      | test3        | 21       | 1         | 70        |
+      | test4        | 14       | 1         | 60        |
+    And the database has the following table 'user_batches':
+      | group_prefix | custom_prefix | size | creator_id |
+      | test         | custom        | 100  | null       |
+      | test         | custom1       | 200  | 13         |
+      | test1        | cust          | 300  | 21         |
+      | test1        | cust1         | 400  | null       |
+      | test2        | cus           | 300  | 21         |
```

Comment:
```suggestion
      | test1        | cust1         | 400  | null       |
```
Maybe having some prefixes without batches would be better for the test.

Reply:
fixed in another way